src/Test/Utilities/Shared/Assert/AssertEx.cs

```diff
@@ -331,55 +331,14 @@ public static void NotNull<T>(T @object, string message = null)
 
         public static void ThrowsArgumentNull(string parameterName, Action del)
         {
-            try
-            {
-                del();
-            }
-            catch (ArgumentNullException e)
-            {
-                Assert.Equal(parameterName, e.ParamName);
-            }
+            var exception = Assert.Throws<ArgumentNullException>(del);
```

Comment:
xunit already has Assert.Throws<ArgumentNullException>(parameterName, Action) that does this.